lib/gateways/resources-gateway.test.js

```diff
@@ -59,7 +59,15 @@ describe('ResourcesGateway', () => {
                 Postcode: 'S1',
                 Tags: ['one', 'two', 'three']
               }
-            }
+            },
+            {
+              id: 'xxx',
+              fields: {
+                supportinInformation: 'This is a description',
+                Prompts: '42 Wallaby Way, Sydney, S1',
+                PromptTag: ['one', 'two', 'three']
```

Comment:
This data doesn't seem to have any assertions against it. Is it just to get the shape of this object right? 

It might not be a problem when seen in context, but I think it might be a smell about putting all these resources together (we talked a little about this on a call!)